test/units/plugins/strategy/test_strategy.py

```diff
@@ -35,6 +35,8 @@
 
 import pytest
 
+pytestmark = pytest.mark.skipif(True, reason="Temporarily disabled due to fragile tests that need rewritten")
```

Comment:
@mattclay @sivel any reason it is not `xfail`? It seems to express the "expected to fail" aspect better.

Reply:
Probably because it segfaults instead of failing.